Gopkg.toml

```diff
@@ -54,6 +54,17 @@ required = [
   # It seems to be broken at HEAD.
   revision = "1ff28809256a84bb6966640ff3d0371af82ccba4"
 
+[[override]]
+  name = "github.com/knative/pkg"
+  # HEAD as of 2018-10-09
+  revision = "ba1a828cd571f605684f9a4506e85bc13cb65b8d"
+
+[[constraint]]
+  name = "github.com/knative/build"
+  # HEAD as of 2018-10-09
+  revision = "0ee769242233fbcff5b36953c55c9cea4f864a9c"
+  source = "github.com/vdemeester/build"
```

Comment:
Remove this line.

Reply:
@ImJasonH that's why I put `[wip]` in the PR title :angel: